src/style.css

```diff
@@ -1,3 +1,72 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+
+  --main-color: #00acdc;
+}
+
+.logo_img {
+  display: block;
+}
+
+.nav_link {
+  text-decoration: none;
+  position: relative;
+  color: black;
+  text-transform: uppercase;
+  height: 15px;
+  display: flex;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 500;
+  padding: 22px 0;
+}
+
+.nav_link_margin {
+  margin-left: 20px;
+}
+
+.nav_list {
+  display: flex;
+  align-items: center;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.is-active::after {
+  /* z-index: 1; */
+  content: '';
+  display: block;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background-color: var(--main-color);
+  position: absolute;
+  left: 0;
+  bottom: -1px;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+header {
+  padding: 0 40px;
```

Comment:
Avoid using tag names for styling, except for `html` and `body`. It's better to use class names or IDs for styling specific elements.